modules/upgrade/pages/migrate-3x-4x.adoc

```diff
@@ -33,7 +34,7 @@ It is important that PostgreSQL 10 is already running on your {productname} 3.
 For more information, see xref:db-migration.adoc[].
 ====
 
-During migration, the database on the source system needs to get exported, or dumped.
+During migration, the database on the source system needs to get dumped.
```

Comment:
I like the term "exported" better than "dumped". A database dump typically is a binary dump, but so it can only be imported into a database with the exactly same version. An export contains the whole database as SQL statements, so importing is much slower, but it works across different database versions. Not really important, but technically experienced users might expect something else when reading "dump".

Reply:
That's a perfectly valid concern!